package.json

```diff
@@ -11,7 +11,6 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/github": "^4.0.0",
-    "@types/console-ui": "^2.2.3",
```

Comment:
This was going to be a separate PR but snuck in here. As long as the tests passed, it's fine. This looks completely unused.